mm2src/common/common.rs

```diff
@@ -634,6 +634,28 @@ pub fn var(name: &str) -> Result<String, String> {
 #[cfg(target_arch = "wasm32")]
 pub fn var(_name: &str) -> Result<String, String> { ERR!("Environment variable not supported in WASM") }
 
+#[cfg(not(target_arch = "wasm32"))]
+pub fn block_on_f01<F>(f: F) -> Result<F::Item, F::Error>
```

Comment:
We need to document this.

Reply:
[1ebc318](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2220/commits/1ebc3180d581aee9a3ec218f03bb1103f9a10724)